### DISCUSSION TITLE
Can't connect after disconnecting

### DISCUSSION THREAD

**Discussion by ManuelSvQ** [winstxnhdw/lc-hax#344] [Q&A]
Whenever I disconnect from a server and try to connect to any other one It gives me the following error. 'Couldn't connect to the server. Are you offline?'
I've tried in fully vanilla LC and it still gives me the same issue.

**Comment by winstxnhdw** [comment]
I can't replicate your issue. Refresh the lobby or restart the game?

**Reply by ManuelSvQ** [reply]
It usually consists of me playing for about an hour with friends and when I try to reconnect to the lobby, or any other lobby after disconnecting it gives me that error, forcing me to restart each time

**Reply by winstxnhdw** [reply]
We haven’t added or changed anything that could provoke this issue. Maybe you could let us know what features you’ve used before this?

**Reply by Dexteriority** [reply]
ONLY when we injected the LC-HAX.
I don't really know what caused it but it only happens if the LC-HAX is injected, i restarted the game and i can connect(without LC-HAX that time).

Mostly this problem occurs when someone invited you through a steam invite, with LC-HAX on, you will got this error everytime you click the link. 
Sometimes it occurs when we're joining public, but its not hard to deal with since we can just click again, but invite links are expired once you click it.

**Comment by o777l** [comment]
This has been happening to me as well and still does. Same error. Now when I join and then somehow get disconnected, I just restart the game because I know it will occur if already injected.

**Reply by winstxnhdw** [reply]
Join lobby first then inject?

**Reply by o777l** [reply]
Yes, that's what I normally do. But as the previous poster mentioned, once out of the server there is an error joining ('Couldn't connect to the server. Are you offline?').

This only happens after injection and is solved by restarting the game

**Reply by winstxnhdw** [reply]
I just ignore those messages and connect again.

**Reply by Dexteriority** [reply]
You can't if its a private game with steam invite.
It'll keep saying "Couldn't connect to the server. Are you offline?" everytime the invite is being accepted

**Reply by winstxnhdw** [reply]
Just right-click their profile and click Join Game?

**Reply by Dexteriority** [reply]
If we're playing on cracked version, no we cant XD

**Reply by winstxnhdw** [reply]
LMFAO. Not sure if it’s a problem I should be trying to solve then since anti-kick heavily relies on Steam ID.

**Comment by winstxnhdw** [comment]
Honestly, I can’t figure out the issue here. There used to be a patch called “TimeoutPatch” that was causing problems for me as well, but it’s been removed for a few days now. If you’re still having this issue, maybe you can try deleting patches until you do not get this problem anymore. Can’t help you more than that.

**Reply by Dexteriority** [reply]
Understood, thank you!
